mcp/src/ui/whoami.module.css

```
@@ -0,0 +1,208 @@
+    --panel-border: rgba(235, 241, 249, 0.1);
+    --panel-shadow: 0 18px 38px rgba(0, 0, 0, 0.34);
+    --text-strong: #f5f7fb;
+    --text-muted: #c1cad7;
+    --accent: #5eead4;
+    --accent-soft: rgba(94, 234, 212, 0.14);
+    --code-bg: rgba(7, 10, 16, 0.96);
+    --code-text: #eef3fb;
+    background:
+      radial-gradient(circle at top left, rgba(45, 212, 191, 0.18), transparent 42%),
+      radial-gradient(circle at top right, rgba(251, 191, 36, 0.16), transparent 35%),
+      linear-gradient(180deg, #0c1219 0%, #111a24 100%);
```

Stylelint (17.9.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)